packages/components/src/components/tools-and-libraries-cards.tsx

```diff
@@ -17,76 +17,76 @@ export function ToolsAndLibrariesCards({ className }: { className?: string }) {
+        {SIX_HIGHLIGHTED_PRODUCTS.map(product => (
+          <AncillaryProductCard key={product.name} as="li" product={product} />
+        ))}
+      </ul>
       <CallToAction href="https://github.com/the-guild-org" variant="primary">
         Explore the Ecosystem
       </CallToAction>
     </section>
   );
 }
 
-function MainCards() {
-  const products = [PRODUCTS.HIVE, PRODUCTS.YOGA, PRODUCTS.MESH, PRODUCTS.CODEGEN];
-  const icons = [HiveIcon, YogaIcon, MeshIcon, CodegenIcon];
+export function MainProductCard({ as: Root, product }: { as: 'div' | 'li'; product: ProductInfo }) {
+  const Decoration = cardDecorations[product.name];
+  const Icon = product.logo;
 
   return (
-    <ul className="-mx-12 -my-2 flex grid-cols-2 flex-row gap-[22px] overflow-auto px-12 py-2 lg:grid xl:grid-cols-4">
-      {products.map((product, i) => {
-        const Icon = icons[i];
-        const Decoration = cardDecorations[product.name];
-        return (
-          <li
-            key={product.name}
-            className="hive-focus-within group relative flex-1 shrink-0 basis-[283.5px] overflow-hidden rounded-2xl bg-blue-400 text-green-1000 first-of-type:bg-green-1000 first-of-type:text-white max-md:w-[283.5px]"
-          >
-            <a className="relative z-10 block flex-1 p-8" href={product.href}>
-              <p className="font-medium">{product.name}</p>
-              <Icon className="mt-8" />
-              <ArrowIcon className="absolute bottom-8 right-8" />
-            </a>
-            <Decoration
-              className="pointer-events-none absolute bottom-0 right-0 fill-blue-200 stroke-[0.5px] opacity-0 transition-opacity duration-500 group-first-of-type:fill-blue-700 group-focus-within:opacity-100 group-hover:opacity-100"
-              preserveAspectRatio="xMidYMid meet"
-            />
-            <HighlightDecoration className="pointer-events-none absolute left-0 top-[-15%] h-[150%] w-full opacity-0 transition-opacity duration-1000 group-focus-within:opacity-100 group-hover:opacity-100" />
-          </li>
-        );
-      })}
-    </ul>
+    <Root
+      key={product.name}
+      className="hive-focus-within group relative flex-1 shrink-0 basis-[283.5px] overflow-hidden rounded-2xl bg-blue-400 text-green-1000 first-of-type:bg-green-1000 first-of-type:text-white max-md:w-[283.5px]"
+    >
+      <a className="relative z-10 block flex-1 p-8" href={product.href}>
+        <p className="font-medium">{product.name}</p>
+        <Icon className="mt-8" />
+        <ArrowIcon className="absolute bottom-8 right-8" />
+      </a>
+      <Decoration
+        className="pointer-events-none absolute bottom-0 right-0 fill-blue-200 stroke-[0.5px] opacity-0 transition-opacity duration-500 group-first-of-type:fill-blue-700 group-focus-within:opacity-100 group-hover:opacity-100"
```

Comment:
```suggestion
      <Decoration
        strokeWidth="0.5"
        className="pointer-events-none absolute bottom-0 right-0 fill-blue-200 opacity-0 transition-opacity duration-500 group-first-of-type:fill-blue-700 group-focus-within:opacity-100 group-hover:opacity-100"
```
to avoid creating class? since it's svg

Reply:
good catch